newIDE/app/src/ProjectsStorage/CloudStorageProvider/CloudResourceMover.js

```diff
@@ -30,94 +38,83 @@ const isBlobURL = (filename: string) => {
   return filename.startsWith('blob:');
 };
 
-export const moveAllCloudProjectResourcesToCloudProject = async ({
-  project,
-  authenticatedUser,
-  oldFileMetadata,
-  newFileMetadata,
-  oldStorageProvider,
-  oldStorageProviderOperations,
-  newStorageProvider,
-  newStorageProviderOperations,
-  onProgress,
-}: MoveAllProjectResourcesOptions): Promise<MoveAllProjectResourcesResult> => {
-  const result: MoveAllProjectResourcesResult = {
-    erroredResources: [],
-  };
-
-  type ResourceToFetchAndUpload = {|
-    resource: gdResource,
-    url: string,
-    filename: string,
-  |};
+const isPrivateAssetUrl = (filename: string) => {
+  return (
+    filename.startsWith('https://private-assets-dev.gdevelop.io') ||
+    filename.startsWith('https://private-assets.gdevelop.io')
+  );
+};
 
-  const newCloudProjectId = newFileMetadata.fileIdentifier;
+/**
+ * Find the resources stored on GDevelop Cloud that must be downloaded and
+ * uploaded into the new project.
+ */
+const getResourcesToFetchAndUpload = ({
+  project,
+  selectCondition,
+  ignoreCondition,
```

Comment:
Not a big fan of these two things, as I can understand what this is selecting or ignoring.
The word "Condition" here gives no clue (sure it's a function, but the typing should tell me this already).